conda/base/context.py

```diff
@@ -821,6 +822,14 @@ def get_help_dict():
             install time. Packages not locally available are downloaded and extracted
             into the first writable directory.
             """),
+        'prompt': dals("""
+            Template for prompt modification based on the active environment. Currently
+            supported template variables are '{prefix}', '{name}', and '{default_env}'.
+            '{prefix}' is the absolute path to the active environment. '{name}' is the
+            basename of the active environment prefix. '{default_env}' holds the value
+            of '{name}' if the active environment is a conda named environment ('-n'
+            flag), or otherwise holds the value of '{prefix}'.
+            """),
```

Comment:
Maybe clarify that this uses `str.format`, e.g., reference https://docs.python.org/3/library/string.html#formatstrings. This would be useful for the user, e.g., if they are greeted with some error when they used some single `{`, `}`, etc.